apis/content/contentapi.yaml

```diff
@@ -2928,3 +3040,30 @@ definitions:
             sourceId:
               type: string
               example: 'copiedId'
+  UpdateCollaboratorsRequest:
```

Comment:
Can you create one UpdateCollaboratorsRequest definition instead of three definitions?